### ISSUE TITLE
Prevent injured from being redrafted with others/Some control outside of military

### ISSUE THREAD

**Issue by FulcrumA** [AnarkisGaming/ATC-Feedback#137]
**Posting here an entry from Discord, as requested:**
Connected to #136 

> Because of Combat toggle mobilizing everyone with military job, injured guys can often leave beds and go soldering while bleeding out. Game defining some characters as critically injured/receiving treatment and excluding them from whole colony mobilisation and a way to manually tell colonists, be they soldiers or civvies to go to bed, a well as manually order doctors to drop things and tend to to the wounded would save one a lot of managing.

Perhaps making heavily injured (with very low HP) but not dead characters/animals - like it is in some similar games - lose consciousness and being a priority salvage for the sake of healing would help

**Comment by IvanBajlo** [comment]
yep old one... people who are bleeding should be rushed to the hospital or even treated on the spot by specialized combat medic (some sort of rare special gear which is very expensive to acquire) ;-)

https://github.com/AnarkisGaming/ATC-Feedback/issues/106

**Comment by SerialKicked** [comment]
I'm aware of the problem. Like everything else in the game, patients "post" jobs, doctor take them, the first one to "post" will generally be the first to be treated. So yep, a hurt toe might occasionally be treated before a lost arm. Changing that would be a lot of trouble on a technical level. So far, I had more urgent things to look at.

> even treated on the spot by specialized combat medic 

Not gonna happen, sadly. 

As for the recruitment thing, I'm never a fan of special rules like "ignoring a skill setting under some condition(s) that only a minor percentage of the player base will ever known about", but yeah, assuming that they have a bleeding wound (HP ticking down) and that their HP is low, I probably should filter them out from the "global" mobilization button.
